components/grid/overview.md

```diff
@@ -161,6 +161,7 @@ The following table lists Grid parameters, which are not discussed elsewhere in
 
 | Parameter | Type and Default&nbsp;Value | Description |
 | --- | --- | --- |
+| `AdaptiveMode` | `AdaptiveMode` enum <br /> (`None`) | Defines the adaptive mode of the Grid. When set to `Auto`, and the window width is below `768px` or `RootComponentAdaptiveSettings.Medium`, components with popups will render them as an `ActionSheet`. |
```

Comment:
You may want to use "adaptive mode" as an anchor leading to an article or a section dedicated to that topic.